.github/workflows/tests.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.7, 3.8 ]
+        python-version: [ 3.7, 3.9 ]
```

Comment:
no tests for 3.8 anymore?

Reply:
we can save some time and assume if it works for the newest and oldest version we support, that stuff in the middle also will work